requirements-min.txt

```
@@ -2,8 +2,9 @@
 h5py==2.10  # support for selection of datasets with list of indices added in 2.10
 importlib-metadata==4.2.0; python_version < "3.8"  # TODO: remove when minimum python version is 3.8
 importlib-resources==5.12.0; python_version < "3.9"  # TODO: remove when when minimum python version is 3.9
-jsonschema==2.6.0
+jsonschema==3.2.0
 numpy==1.16  # numpy>=1.16,<1.18 does not provide wheels for python 3.8 and does not build well on windows
 pandas==1.0.5  # when this is changed to >=1.5.0, see TODO items referenced in #762
 ruamel.yaml==0.16
 scipy==1.1  # scipy>=1.1,<1.4 does not provide wheels for python 3.8 and building scipy can fail due to incompatibilities with numpy
+linkml-runtime==1.5.0
```

```suggestion
linkml-runtime==1.5.0
tqdm==4.41.0
zarr==2.12.0
```
By convention, we have not been testing the minimum versions of optional dependencies. We can do so, but for consistency, we should also do so for tqdm and zarr.